### PR TITLE
feat(rating): round rating display to nearest decimal, not quarter

### PR DESCRIPTION
**Pull request by cowills**
https://issues.rei.com/browse/CDR-1917

### CODE REVIEW

File: src/components/rating/examples/Ratings.vue [hunk 1]
```diff
@@ -60,7 +74,7 @@
     />
     <div class="line-height-reset">
       <cdr-rating
-        rating="3.66"
+        rating="3.63"
```
Comment (mhewson) [inline src/components/rating/examples/Ratings.vue:77]: is there still a need for the rounded prop or should everything be moved to the displayRating - UGC is only interested in the 10's being displayed

File: src/components/rating/__tests__/CdrRating.spec.js [hunk 2]
```diff
@@ -22,27 +23,31 @@ describe('CdrRating', () => {
     expect(wrapper.vm.whole).toBe(3);
```
Comment (mhewson) [inline src/components/rating/__tests__/CdrRating.spec.js:23]: per the UGC convo today, they are interested in having the decimal place always present - 3.0 vs 3

File: src/components/rating/examples/Ratings.vue (hunk 1 above)
Comment (cowills) [reply src/components/rating/examples/Ratings.vue:77]: yep we need that to figure out how many stars to show

File: src/components/rating/__tests__/CdrRating.spec.js (hunk 2 above)
Comment (mhewson) [reply src/components/rating/__tests__/CdrRating.spec.js:23]: and I'm not 100% sure but I believe the ask would be to move from .75 to .8 @NEWeber can you confirm?

Comment (cowills) [reply src/components/rating/__tests__/CdrRating.spec.js:23]: already did the update from `.75` to `.8`. Just pushed a commit that makes `3` render as `3.0`

**Review by mhewson — APPROVED** [review]

File: src/components/rating/__tests__/CdrRating.spec.js (hunk 2 above)
Comment (NEWeber) [reply src/components/rating/__tests__/CdrRating.spec.js:23]: > and I'm not 100% sure but I believe the ask would be to move from .75 to .8 @NEWeber can you confirm?

Yes, that's right

**Pull request merged**